### DISCUSSION TITLE
Is EFCore not compatible with Uno Mobile?

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#9113] [Q&A] [answered]
/usr/local/share/dotnet/packs/Microsoft.MacCatalyst.Sdk/15.4.311/targets/Xamarin.Shared.Sdk.targets(1219,3) error: clang++ exited with code 1. This occurs even using EFCore 6 which supports net6.0-macatalyst

Also by default Uno generates with Microsoft.Extensions.Logging 5.0.0 and Microsoft.Extensions.Logging.Console while EFCore for compatibility requires 6.0.0. Are there plans to increase this default template setting?

**Comment by jeromelaban** [comment, marked as answer]
EF Core needs support for .NET 6 Mobile, and https://github.com/dotnet/efcore/issues/28117 looks like a similar issue.

**Reply by bstrother1** [reply]
Should this discussion be linked? Also the nuget [nuget.org/packages/Microsoft.EntityFrameworkCore/6.0.6#supportedframeworks-body-tab](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore/6.0.6#supportedframeworks-body-tab) has net6.0-maccatalyst listed as :  Additional computed target framework(s), which should mean implicit support. It seems like listing a package as supporting net6 doesn't actually mean supporting net6, just windows.

**Reply by jeromelaban** [reply]
The packages do seem to support it, but if the build fails, it's something you'll have to report to the EF Core team, Uno can't do much about that part.
